packages/icons/src/PixivIcon.ts

```diff
@@ -190,23 +195,28 @@ export class PixivIcon extends HTMLElement {
     width: var(--size);
     height: var(--size);
   }
-</style>`,
-      { ALLOWED_TAGS: ['style'], FORCE_BODY: true }
-    )
+</style>`
 
-    const svg = DOMPurify.sanitize(
+    const svg =
       this.svgContent !== undefined
         ? this.svgContent
-        : `<svg viewBox="0 0 ${size} ${size}"></svg>`,
-      { USE_PROFILES: { svg: true, svgFilters: true } }
-    )
+        : `<svg viewBox="0 0 ${size} ${size}"></svg>`
 
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     this.shadowRoot!.innerHTML = style + svg
   }
 
   private async loadSvg(name: string) {
-    this.svgContent = await getIcon(name)
+    const iconResult = await getIcon(name)
+
+    if (iconResult.trusted) {
+      this.svgContent = iconResult.svgContent
+    } else {
+      const { default: DOMPurify } = await import('dompurify')
```

Comment:
[dom-purify](https://bundlephobia.com/package/dompurify@3.0.5) is still pretty large so

1. let user do sanitizing themselves
2. leveraging https://developer.mozilla.org/en-US/docs/Web/API/HTML_Sanitizer_API